libs/utils/utils.js

```diff
@@ -461,7 +461,13 @@ export function decorateImageLinks(el) {
 export function decorateAutoBlock(a) {
   const config = getConfig();
   const { hostname } = window.location;
-  const url = new URL(a.href);
+  let url;
+  try {
+    url = new URL(a.href);
+  } catch (e) {
+    window.lana?.log(`Cannot make URL from decorateAutoBlock - ${a?.href}: ${e.toString()}`);
+  }
+  if (!url) return false;
```

Comment:
nit: save a branch by returning in catch. new URL() will not return null: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL